mint-auditor/src/http_api/api_types.rs

```diff
@@ -0,0 +1,59 @@
+use crate::db::{Counters, MintAuditorDb};
+use ::std::collections::HashMap;
+use rocket::serde::Serialize;
+
+/// type for auditor db in Rocket state
+pub struct AuditorDb(pub MintAuditorDb);
```

Comment:
Why not pass `MintAuditorDb` directly?

Reply:
Was following along with the rocket docs, which show wrapping state in a custom struct, but there's no reason it has to be done that way for this state. I'll change it